custom_components/tuya_local/devices/holman_wx8_irrigation_controller.yaml

```diff
@@ -174,6 +174,17 @@ secondary_entities:
           - dps_val: 8
             value: true
           - value: false
+  - entity: binary_sensor
```

Comment:
I would not include this, or if you want it for observation, make it an attribute of one of the other entities rather than a standalone sensor

Reply:
I put this in when I was testing the mapping. I will remove it. I don't believe it means anything.